lib/dialectInspect/mysql.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+var Promise = require("bluebird")
+  , _ = require("lodash")
+  , path = require("path")
+
+  , Migration = require("../Migration")
+  , SchemaCatalog = require("../schemaCatalog")
+  , utils = require("../utils");
+
+var _catalog = "demo"
+  , constants = utils.constants;
+
+var _client
+  , _statements;
+
+module.exports.listTables = function (client) {
+  return _setClient(client)
+    .then(function(result) {
+      _catalog = result[0][0]['database()'];
```

Comment:
@hmunn you'll want to replace these calls w/ lodash; e.g.
```
_.get(result, [0, 0, "database()"], <defaultValue>); 
```
This will avoid an undefined reference error.